xlsynth/tests/sv_bridge_test.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use pretty_assertions::assert_eq;
```

Comment:
considering patching this out, in our internal copy would that be fine with you?

Reply:
/cc @cdleary